src/content/React-CSharp/ClientApp/src/components/NavMenu.js

```diff
@@ -1,56 +1,42 @@
-import React, { Component } from 'react';
+import React from 'react';
+import { useState } from 'react';
 import { Collapse, Container, Navbar, NavbarBrand, NavbarToggler, NavItem, NavLink } from 'reactstrap';
 import { Link } from 'react-router-dom';
 ////#if (IndividualLocalAuth)
 import { LoginMenu } from './api-authorization/LoginMenu';
 ////#endif
 import './NavMenu.css';
 
-export class NavMenu extends Component {
-  static displayName = NavMenu.name;
+export function NavMenu() {
+  const [collapsed, setCollapsed] = useState(true);
 
-  constructor (props) {
-    super(props);
+  const toggleNavbar = () => setCollapsed(!collapsed);
```

Comment:
nit: this should also use an updater function [[docs]](https://reactjs.org/docs/faq-state.html#how-do-i-update-state-with-values-that-depend-on-the-current-state)

```suggestion
  const toggleNavbar = () => setCollapsed(current => !current);
```